### PR TITLE
Create LICENSE.md

### LINKED ISSUE #38: License?
Jake mentioned the `container-ontology` was under the Apache license (https://forums.pylabrobot.org/t/database-storing-common-labware-dwps-etc-specifications/103/15), but I don't see a License.md in this repo?

### PR DESCRIPTION
**Pull request by photocyte**
Add Apache 2.0 LICENSE.md with "2022 Bioprotocols Working Group"

Closes #38

### CODE REVIEW

**Comment by rpgoldman** [comment]
What did you mean by

> with "2022 Bioprotocols Working Group

I don't see that in the LICENSE.md file

Thanks!

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [2022] [Bioprotocols Working Group]
```
Comment (photocyte) [inline LICENSE.md:189]: Highlighting the "2022 Bioprotocols Working Group" section

**Comment by rpgoldman** [comment]
In the interests of credit, I would prefer this be “Robert P. Goldman; SIFT, LLC; and the Bioprotocols Working Group.”

**Comment by jakebeal** [comment]
@photocyte Given the right now, the only person who has made significant commits in this repository is @rpgoldman , I think that would be reasonable. 

Once there are more contributors, I would suggest moving to one of the standard opensource conventions like "The Bioprotocols Working Group and all contributors in the CONTRIBUTORS.md file"

**Comment by photocyte** [comment]
Understood. I have made the requested changes from @rpgoldman . Do you want to do the honors to merge it @rpgoldman ? Will leave setting up a CONTRIBUTERS.md to future deliberation.

**Review by danbryce — APPROVED** [review]

**Pull request merged**